plugin/core/main.py

```diff
@@ -217,17 +217,80 @@ def start_client(window: sublime.Window, config: ClientConfig):
         "rootPath": project_path,
         "capabilities": {
             "textDocument": {
+                "synchronization": {
+                    "didSave": True
+                },
+                "hover": {
+                    "contentFormat": ["plaintext"]
+                },
                 "completion": {
                     "completionItem": {
-                        "snippetSupport": True
+                        "snippetSupport": True,
+                        "documentationFormat": ["plaintext"]
+                    },
+                    "completionItemKind": {
+                        "valueSet": [
+                            CompletionItemKind.Text,
+                            CompletionItemKind.Method,
+                            CompletionItemKind.Function,
+                            CompletionItemKind.Constructor,
+                            CompletionItemKind.Field,
+                            CompletionItemKind.Variable,
+                            CompletionItemKind.Class,
+                            CompletionItemKind.Interface,
+                            CompletionItemKind.Module,
+                            CompletionItemKind.Property,
+                            CompletionItemKind.Unit,
+                            CompletionItemKind.Value,
+                            CompletionItemKind.Enum,
+                            CompletionItemKind.Keyword,
+                            CompletionItemKind.Snippet,
+                            CompletionItemKind.Color,
+                            CompletionItemKind.File,
+                            CompletionItemKind.Reference
+                            ]
                     }
                 },
-                "synchronization": {
-                    "didSave": True
-                }
+                "signatureHelp": {
+                    "signatureInformation": {
+                        "documentationFormat": ["plaintext"]
```

Comment:
I'm fairly certain you can set this to `markdown`.

Reply:
I've added markdown to the list.